tests/integration/simple_integration_test.go

```diff
@@ -0,0 +1,164 @@
+package integration_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+var (
+	namespace = "k8ssandra"
+	releaseName string
+	releaseNameCluster string
+)
+
+var _ = Describe("Install the cluster", func() {
+	// TODO This assumes the kind was started with correctly setup cluster. Could we automate it here to provider proper e2e?
+
+	BeforeSuite(func() {
+		kubectlOptions := k8s.NewKubectlOptions("", "", "default")
+		options := &helm.Options{KubectlOptions: kubectlOptions}
+
+		// Add traefik repo and update repos
+		helm.RunHelmCommandAndGetOutputE(GinkgoT(), options, "repo", "add", "traefik", "https://helm.traefik.io/traefik")
+		helm.RunHelmCommandAndGetOutputE(GinkgoT(), options, "repo", "update")
+
+		// Deploy traefik
```

Comment:
I am fine with leaving traefik out for an initial set of tests. In fact, I prefer to leave it out and tackle that separately.

Reply:
Then we need some other method of accessing the cluster resources (such as prometheus). A port-forwarding is another option, but that doesn't really test if we actually have in-cluster working solution or just in-pod.